ui/app/components/auth/form/base.ts

```diff
@@ -57,16 +64,27 @@ export default class AuthBase extends Component<Args> {
           selectedAuth: this.args.authType,
         });
 
-        this.handleAuthResponse(authResponse, this.args.authType, formData);
+        const path = formData?.path;
+        this.handleAuthResponse(authResponse, path);
       } catch (error) {
         this.onError(error as Error);
       }
     })
   );
 
-  handleAuthResponse(authResponse: AuthData, authType: string, formData?: object) {
+  // Standard methods get mfa_requirements from the authenticate method in the auth service
```

Comment:
Long term plan is that each method component no longer call `this.auth.authenticate` and instead uses the api service for to explicitly make its login request. Once the auth logic isn't so abstracted handling MFA can be consolidated to one place (either here or the auth service) because we will have more control over the authResponse.